.github/workflows/docker-stack-sipnet.yml

```
@@ -1,13 +1,11 @@
 name : Docker Stack Model Test
 on :
-  pull_request:
-
   # allow manual triggering
   workflow_dispatch:
 
   schedule:
     # run Thursday 4:30 AM UTC
-  - cron: '30 4 * * 4'    
+  - cron: '30 4 * * */3'
```

According to the PR description, “once every 3 days” would correspond to: `30 4 */3 * *`

The current cron: `30 4 * * */3` runs on sun, wed, and sat, which is not always once every 3 days.

Did you intend to schedule it on specific weekdays (Sun/Wed/Sat)?
If so, the current cron is correct; otherwise, the schedule may need to be adjusted.

---

Good catch! Let's stick to specific days of the week but make more of them land on weekdays. How does Sunday/Tuesday/Thursday sound?
```suggestion
  - cron: '30 4 * * 0,2,5'
```

---

Yes, that works. One small nit: instead of running it on Sunday(i.e. 0) , what if we use `30 4 * * 1,3,5`, i.e., Monday, Wednesday, and Friday (all weekdays)?

---

Works for me!

(I could come up with reasons to prefer one weekday over another, but we should recognize this as classic [bikeshedding](https://en.wikipedia.org/wiki/Law_of_triviality). Any schedule that runs "often enough we can figure out what changed since last time" will work here.)
```suggestion
  - cron: '30 4 * * 1,3,5'
```